pkgs/tools/misc/hyperledger-fabric/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "hyperledger-fabric-${version}";
```

```suggestion
  pname = "hyperledger-fabric";
```
`buildGoPackage` can have just a `pname` and a `version` now.

---

I forgot about this, fixed. Thanks!